databases-analytics/databases/postgresql-tf/README.md

```diff
@@ -0,0 +1,34 @@
+## Create PostgreSQL Database with Terraform
+
+### General information
+ - 🔗 [Using Terraform with OVHcloud](https://help.ovhcloud.com/csm/fr-terraform-at-ovhcloud?id=kb_article_view&sysparm_article=KB0054776)
+ - 🔗 [Find out how to order and manage your Public Cloud managed database service using Terraform ](https://help.ovhcloud.com/csm/fr-public-cloud-databases-order-terraform?id=kb_article_view&sysparm_article=KB0048840)  
+ - 🔗 [How to use Terraform](https://help.ovhcloud.com/csm/en-gb-public-cloud-compute-terraform?id=kb_article_view&sysparm_article=KB0050787)
+ - 🔗 [cloud_project_database](https://registry.terraform.io/providers/ovh/ovh/latest/docs/resources/cloud_project_database)
+ - 🔗 [OVH token generation page](https://www.ovh.com/auth/api/createToken?GET=/*&POST=/*&PUT=/*&DELETE=/*)
+ - 🔗 [public-cloud-databases-examples](https://github.com/ovh/public-cloud-databases-examples)
+
+### Set up
+  - Install the [Terraform CLI](https://www.terraform.io/downloads.html)
+  - Get the credentials from the OVHCloud Public Cloud project:
+    - `endpoint`
+    - `application_key`
+    - `application_secret`
+    - `consumer_key`
+  - Get the `service_name` (Public Cloud project ID)
+  - Get your public IP address (you can use https://whatismyipaddress.com/ for example)
```

Comment:
```suggestion
  - Get your public IP address (you can use https://whatismyipaddress.com/ or curl ifconfig.me for example)
```

Reply:
sorry I missed the `ifconfig.me` part in your previous review, my bad.

I added it now with: https://github.com/ovh/public-cloud-examples/pull/38/commits/411b8c1ebb62959c2a2fb4c963180c308b5ae94f